src/app/apis/details.js

```diff
@@ -0,0 +1,44 @@
+const LOAD = 'reactcapstone/apis/LOAD';
+
+const init = [];
+
+export function load(rendered) {
+  return {
+    type: LOAD,
```

Comment:
- [ ] please ensure that each card has its separate information in the details page. From the details page of one of your cards, i can scroll and see all the information of the other cards. Please fix that.

![demo33](https://user-images.githubusercontent.com/65298048/195948281-39a2cca1-4613-446e-9c0c-0af80dd8da39.gif)